src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -418,6 +437,9 @@ export class StudyViewPageStore {
                 }
             }
         );
+        this.driverAnnotationSettings = buildDriverAnnotationSettings(
```

Comment:
move this up to property def

Reply:
Moved. Problem is that this cannot be done in OncoprinterStore.ts:

```
    @observable driverAnnotationSettings: DriverAnnotationSettings;
    ...
    private initialize() {
        this.driverAnnotationSettings = buildDriverAnnotationSettings(
            () => this.didOncoKbFail
        );
```
I need to re-implement this after rebase to master.